playground/cdk/CdkSample.AppHost/aspire-manifest.json

```diff
@@ -27,214 +27,4 @@
-        "principalType": ""
-      }
-    },
-    "signalr": {
-      "type": "azure.bicep.v0",
-      "connectionString": "Endpoint=https://{signalr.outputs.hostName};AuthType=azure",
-      "path": "signalr.module.bicep",
-      "params": {
-        "principalId": "",
-        "principalType": ""
-      }
-    },
-    "logAnalyticsWorkspace": {
-      "type": "azure.bicep.v0",
-      "path": "logAnalyticsWorkspace.module.bicep"
-    },
-    "appInsights": {
-      "type": "azure.bicep.v0",
-      "connectionString": "{appInsights.outputs.appInsightsConnectionString}",
-      "path": "appInsights.module.bicep",
-      "params": {
-        "logAnalyticsWorkspaceId": "{logAnalyticsWorkspace.outputs.logAnalyticsWorkspaceId}"
-      }
-    },
-    "api": {
-      "type": "project.v0",
-      "path": "../CdkSample.ApiService/CdkSample.ApiService.csproj",
-      "env": {
-        "OTEL_DOTNET_EXPERIMENTAL_OTLP_EMIT_EXCEPTION_LOG_ATTRIBUTES": "true",
-        "OTEL_DOTNET_EXPERIMENTAL_OTLP_EMIT_EVENT_LOG_ATTRIBUTES": "true",
-        "ASPNETCORE_FORWARDEDHEADERS_ENABLED": "true",
-        "ConnectionStrings__signalr": "{signalr.connectionString}",
-        "ConnectionStrings__blobs": "{blobs.connectionString}",
-        "ConnectionStrings__sqldb": "{sqldb.connectionString}",
-        "ConnectionStrings__mykv": "{mykv.connectionString}",
-        "ConnectionStrings__cache": "{cache.connectionString}",
-        "ConnectionStrings__cosmos": "{cosmos.connectionString}",
-        "ConnectionStrings__pgsqldb": "{pgsqldb.connectionString}",
-        "ConnectionStrings__servicebus": "{servicebus.connectionString}",
-        "ConnectionStrings__appConfig": "{appConfig.connectionString}",
-        "ConnectionStrings__search": "{search.connectionString}",
-        "APPLICATIONINSIGHTS_CONNECTION_STRING": "{appInsights.connectionString}"
-      },
-      "bindings": {
-        "http": {
-          "scheme": "http",
-          "protocol": "tcp",
-          "transport": "http",
-          "external": true
-        },
-        "https": {
-          "scheme": "https",
-          "protocol": "tcp",
-          "transport": "http",
-          "external": true
-        }
-      }
-    },
-    "pgsql2-username": {
-      "type": "parameter.v0",
-      "value": "{pgsql2-username.inputs.value}",
-      "inputs": {
-        "value": {
-          "type": "string",
-          "default": {
-            "generate": {
-              "minLength": 10,
-              "numeric": false,
-              "special": false
-            }
-          }
-        }
-      }
-    },
-    "pgsql2-password": {
-      "type": "parameter.v0",
-      "value": "{pgsql2-password.inputs.value}",
-      "inputs": {
-        "value": {
-          "type": "string",
-          "secret": true,
-          "default": {
-            "generate": {
-              "minLength": 22
-            }
-          }
-        }
-      }
-    }
-  }
-}
\ No newline at end of file
+      "type": "azure.bicep.v0"
```

Comment:
this seems incomplete.

Reply:
Yeah ... draft because we are waiting for a fix from the Azure SDK team. KeyVault and Storage are throwing when we set their firewall rules hence the incomplete output here. Once I get the updated bits I'll regen all of this.